test/services/model.spec.js

```diff
@@ -14,11 +14,19 @@ describe('Model library', () => {
       label: [vocabulary[3], vocabulary[4], vocabulary[5]].join(' '),
       cozyCategoryId: '200'
     },
+    {
+      label: [vocabulary[3], vocabulary[5], vocabulary[5]].join(' '),
+      cozyCategoryId: '200'
+    },
     {
       label: [vocabulary[3], vocabulary[1], vocabulary[2]].join(' ')
     },
     {
       label: [vocabulary[6], vocabulary[5], vocabulary[4]].join(' ')
+    },
+    {
+      label: 'bgsdhgisuh bghebgo gjnpjrgpgnjezjn vnsdflkjnio',
```

Comment:
Is it a cat-generated operation? :cat2:

Reply:
I'm allergic to cats, it's elbow-generated!